js/game.js

```diff
@@ -5,8 +5,75 @@ export default class Game {
     this.tamagotchi = new Tamagotchi();
   }
 
-  start = ({ healthElement, hungerElement, energyElement, funElement }) => {
-    this.tamagotchi.mount({ healthElement });
+  start = ({
+    healthElement,
+    hungerElement,
+    energyElement,
+    funElement
+  }) => {
+    this.tamagotchi.mount({
+      healthElement,
+      hungerElement,
+      energyElement,
+      funElement,
+    });
+
     console.log("Game started");
+
+    const decreaseHealth = () => {
```

Comment:
**nitpick:** Let's move out these methods (`decreaseHealth`, `decreaseHunger`, etc.) to the class body - or to Tamagotchi class. It will be easier for you to reuse these methods later. Now you can't really do anything with them because they are bound to the `start` method. Try to think of it, as you would like to have a possibility to run `decreaseHealth` independently. For example: `decreaseHealth(pointsToDecrease) { ... }`

Reply:
I agree - it works, but it's not maintainable, and reusable code yet. I see that you may not be familiar with `this` keyword. For example - you can extend your Tamagotchi parameters constructor so it keeps `elements` that you catch for displaying values (screen 1), then you access it from your methods (screen 2). In this example I extracted it based on your current code

<img width="597" alt="Zrzut ekranu 2023-02-28 o 13 37 15" src="https://user-images.githubusercontent.com/22893188/221856011-eecb82b6-b87b-4380-bdfd-3b18fc775aff.png">

<img width="725" alt="Zrzut ekranu 2023-02-28 o 13 37 35" src="https://user-images.githubusercontent.com/22893188/221856110-b150c8e6-7a6e-4fc6-8e97-2970e82df8f4.png">